ubo_app/services/030-wifi/setup.py

```diff
@@ -120,6 +90,36 @@ async def _check_connection() -> None:
             'saved_wifi_ssids': await get_saved_wifi_ssids(),
         },
     )
+    onboarding_notification = Notification(
+        title='No internet connection',
+        content='Press middle button "󱚾" to add WiFi network',
+        importance=Importance.MEDIUM,
+        icon='󱚵',
+        display_type=NotificationDisplayType.STICKY,
+        actions=[
+            NotificationActionItem(
+                action=lambda: (
+                    create_wireless_connection.CreateWirelessConnectionPage
+                ),
+                icon='󱚾',
+                background_color=INFO_COLOR,
+                dismiss_notification=True,
+            ),
+        ],
+        extra_information=ReadableInformation(
+            text='Press middle button to add WiFi network with QR code.\n'
```

Comment:
```suggestion
            text='Press middle button to add a WiFi connection.\n'
```
@sassanh This message is a bit outdated (comes from the time we had support for QR code only)